TestResultSummaryService/parsers/Test.js

```diff
@@ -111,7 +122,8 @@ class Test extends Parser {
         }
         return {
             tests: results,
-            type: "Test"
+            type: isPerf ? "Perf" : "Test",

```

Comment:
I think we might run into some issues if we run both perf and functional tests in one Jenkins job. Even though we'll be able to parse both and store perf results for just perf tests when both are run, it might be an issue for Perf Test view. 

@llxia Any suggestions? I would recommend just going with this for now and enhancing this in future PRs.